lib/gs-plugin-vfuncs.h

```diff
@@ -631,8 +631,8 @@ gboolean	 gs_plugin_app_copy			(GsPlugin	*plugin,
  * @cancellable: a #GCancellable, or %NULL
  * @error: a #GError, or %NULL
  *
- * Determine whether gs_plugin_os_copy() with this @app and @copy_dest should
- * succeed (barring runtime errors which cannot be known in advance).
+ * Determine whether gs_plugin_os_copy() with this @copy_dest should succeed
+ * (barring runtime errors which cannot be known in advance).
  *
  * Returns: %FALSE if copying the OS to @copy_dest will certainly fail; %TRUE
  * otherwise
```

Comment:
I was meaning that the `Returns` documentation seems to be about `copyable`, rather than whether determining the copyability failed.

Reply:
Ah, got it. I fixed this and the same mistake for `gs_plugin_app_get_copyable()`, rebased on master, and force-pushed. Please re-review.